### DISCUSSION TITLE
Overview of sentences and improving widgets

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#769] [Ideas]
@sonvnn I have a site on Joomla 3, Astroid 2 and JD -Builder.
He began to transfer to the last Joomla and Astroid. I collected the old site using widgets. It seems not bad, as in the original, but there is a remark.

1) Button - it would be great if it was possible to add the icon.
2) Heading - there is an addition of icons, but there is no opportunity to configure it, indicate the size, add your icon, etc.
3) Grid - I wanted to create a personnel block, there is no opportunity to add social network, the text editor does not understand HTML and it doesn’t have the opportunity to create social networks
4) Grid - I wanted to create a block of services by adding my icon in PNG format, but could not change the size of the pictures. There is no opportunity
5) List - the code should be observed by UL and Li, but there is h instead of tanning.
And it would be great there to add a link to tanks as in Heading and control of icons, change color and dimensions
6) Module Position - it's great that you can choose a position module but if you add a choice of a separate module, it would be super
7) Form Builder - It would be nice to add to change the text of the Send button
8) Testimonials - there is no possibility of managing avatar, add retreats of Margin and Padding

But in general, I am satisfied with a bang.
What was on my old site was able to assemble again with the help of widgets. Huge saved you for your work

If there are mistakes, sorry, this is all Googol translator

**Comment by rinka88** [comment]
@sonvnn I think it's worth watching these improvements

**Comment by sonvnn** [comment]
Thank you! I will review and add them to tasklist

**Comment by Dnsftl** [comment]
Hi @sonvnn,

Would like to add more to the list, if it is possible.

- An option to enable link adributs such as _self, _blank, ect in the text editor.
- Another option to view/edit the source code in the text editor
- In the accordion widget to have an option to have all fields closed, instead of the first one always being open
- In sociaal delen add Instagram en Threads? 

Hope it is possible, it would help us a lot

Thank you in advance

Regards, 

Dennis

**Comment by sonvnn** [comment]
@rinka88 @Dnsftl I just improved Button Widget and Accordion Widget.

- Button Widget: Support display icon on button
- Accordion Widget: Add option close all field and option always open.

Let's download v3-branch and check it.

**Reply by Dnsftl** [reply]
Hi Sonny,

Do you have a link to the new v3-branch?

**Reply by sonvnn** [reply]
You can download it here https://github.com/templaza/astroid-framework/archive/refs/heads/v3.zip

**Reply by Dnsftl** [reply]
Thank you

**Comment by rinka88** [comment]
@sonvnn 6. Will it be possible to select a specific module rather than a whole module position?

**Reply by rinka88** [reply]
don't forget this request))

**Comment by rinka88** [comment]
9. in the Slideshow widget, add another button - a link

**Comment by rinka88** [comment]
10.Add several amounts of styles to the divider, not like now 3 types

**Comment by rinka88** [comment]
11. You probably need to add languages and ACL in the widget settings
So that you can configure it by site language and display it by user groups
@sonvnn

**Reply by rinka88** [reply]
> 11. You probably need to add languages and ACL in the widget settings
>     So that you can configure it by site language and display it by user groups
>     @sonvnn

I think we also need to look at these improvements. Since these functions are needed for a multilingual site and more complex ones for which user access levels are required

**Reply by rinka88** [reply]
+
